crates/noirc_driver/src/lib.rs

```diff
@@ -399,6 +425,6 @@ impl Driver {
 impl Default for Driver {
     fn default() -> Self {
         #[allow(deprecated)]
-        Self::new(&Language::R1CS, Box::new(acvm::pwg::default_is_opcode_supported(Language::R1CS)))
+        Self::new()
     }
 }
```

Comment:
Let's derive this and then have `Driver::new()` point at `Driver::default()` (or remove it).

At the very least we should remove the `#[allow(deprecated)]` as it's stale.